doc/error_codes.md

```diff
@@ -0,0 +1,4 @@
+# Latest Used Error Codes
+
+E-VS-COM-JDBC-31
```

Comment:
Cool idea.

Should we use "latest" or "next" ?

Anyway, looks like the last one use is `E-VS-COM-JDBC-33`.